app/components/listing/feedback/FeedbackSteps.jsx

```diff
@@ -52,17 +52,15 @@ export const FeedbackTags = ({ tagOptions, onSelectTag }) => (
   </div>
 );
 
-export const Review = ({ reviewValue, onReviewChange, isReviewRequired }) => (
+export const Review = ({ reviewValue, onReviewChange }) => (
   <div className={styles.feedbackReview}>
     <div className={styles.stepsPrompt}>
       Please provide your feedback below:
     </div>
     <textarea
       className={styles.feedbackTextarea}
       type="text"
-      placeholder={`Type your feedback here ${
-        !isReviewRequired ? '(optional)' : ''
-      }`}
+      placeholder="Type your feedback here (optional)"
```

Comment:
I'm a little bit confused by the screenshot you included in your PR description. The screenshot shows a placeholder text of "Type your feedback here (required)," but neither the old code nor the new code show that text. Was that screenshot taken from some intermediate version of the code with the incorrect text?

Just wanted to double check that there isn't some other place in the code that determines the placeholder text shown in that screenshot, because if there is, we should update that text not not include the word "required."

Reply:
Just fixed it, didn't realized I uploaded wrong screenshot